### PR TITLE
Add prompt for Flutter 2019 Q3 survey

### PR DESCRIPTION
**Pull request by DanTup**
FYI @pq @jayoung-lee

### CODE REVIEW

File: src/shared/vscode/user_prompts.ts [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
+import { Context } from "./workspace";
 
-// Mon May 13 2019 20:00:00 GMT+0100 (BST) = noon PDT on 13th May
-export const surveyStart = Date.UTC(2019, 4 /* Month is 0-based!! */, 13, 19, 0);
-// Mon May 27 2019 08:00:00 GMT+0100 (BST) = midnight PDT between 26th/27th may.
-export const surveyEnd = Date.UTC(2019, 4 /* Month is 0-based!! */, 27, 7, 0);
+// Mon Aug 12 2019 17:00:00 GMT+0100 (British Summer Time UTC+1) = Mon Aug 12 09:00 PDT (UTC-7)
+export const surveyStart = Date.UTC(2019, 7 /* Month is 0-based!! */, 12, 16, 0);
+// Sun Aug 25 2019 02:00:00 GMT+0100 (British Summer Time UTC+1) = Sat Aug 24 18:00 PDT (UTC-7).
+export const surveyEnd = Date.UTC(2019, 7 /* Month is 0-based!! */, 25, 1, 0);
```
Comment (DanTup) [inline src/shared/vscode/user_prompts.ts]: @jayoung-lee please double-check these dates. They're written as UTC and to confuse things, months are 0-based and the end date is "the next  day" in UTC.

The first parts of the comments (the time written in BST) are the output of printing the date from the next line in a JS console (though the conversions after the `=` are my own!).

File: src/shared/vscode/user_prompts.ts [hunk 2]
```diff
@@ -48,22 +48,22 @@ export function showFlutter2019Q2SurveyNotificationIfAppropriate(context: Contex
 		logger.warn("Unable to read Flutter settings for preparing survey link");
 	}
 
-	const prompt = clientID ? flutterSurvey2019Q2PromptWithAnalytics : flutterSurvey2019Q2PromptWithoutAnalytics;
-	const surveyUrl = "https://google.qualtrics.com/jfe/form/SV_3W3aVD2y9CoAe6V?Source=VSCode"
+	const prompt = clientID ? flutterSurveyPromptWithAnalytics : flutterSurveyPromptWithoutAnalytics;
+	const surveyUrl = "https://google.qualtrics.com/jfe/form/SV_3kiGXYfYOfXUjB3?Source=VSCode"
```
Comment (DanTup) [inline src/shared/vscode/user_prompts.ts]: @jayoung-lee also double-check the URL. Your example has "IDE", but I stuck with "VSCode".

File: src/shared/vscode/user_prompts.ts (hunk 1 above)
Comment (jayoung-lee) [reply src/shared/vscode/user_prompts.ts]: LGTM :)

File: src/shared/vscode/user_prompts.ts (hunk 2 above)
Comment (jayoung-lee) [reply src/shared/vscode/user_prompts.ts]: The link is correct!

File: src/shared/vscode/user_prompts.ts (hunk 1 above)
Comment (DanTup) [reply src/shared/vscode/user_prompts.ts]: Thanks!

**Comment by DanTup** [comment]
Test failures are unrelated - working on them on `master`.

**Pull request merged**